_includes/mobile/intro-prebid-rendered.md

```diff
@@ -0,0 +1,128 @@
+- **GAM** and the **Google Mobile Ads (GMA) SDK** manage the ad inventory and select the winning ad to display.
+- **Prebid SDK** renders display ads when a Prebid bid wins.
+- **GMA SDK** renders video ads when a Prebid bid wins.
+
+If you do not have GMA SDK in the app yet, refer to the [Google Integration Documentation](https://developers.google.com/ad-manager/mobile-ads-sdk/{{include.platform}}/quick-start).
+
+### Alternative Approaches
+
+Another way to integrate GAM into your app is with the [GAM Bidding-Only integration](/prebid-mobile/pbm-api/{{include.platform}}/{{include.platform}}-sdk-integration-gam-original-api.html).
+
+Tradeoffs between these integration approaches:
+
+{: .table .table-bordered .table-striped }
+| Aspect | Bidding-Only Integration | Prebid-Rendered Integration |
+| --- |:---:|:---:|
+| App code has direct access to bids | <img alt="check" src="/assets/images/icons/icon-check-green.png" width="30"> | |
+| Support for MRAID 3.0 | | <img alt="check" src="/assets/images/icons/icon-check-green.png" width="30"> |
+| Support for SKAdnetwork | | <img alt="check" src="/assets/images/icons/icon-check-green.png" width="30"> |
+| Loads data from Prebid Cache | <img alt="check" src="/assets/images/icons/icon-check-green.png" width="30"> | |
+| Supports instream video | <img alt="check" src="/assets/images/icons/icon-check-green.png" width="30"> | |
+| Triggers billing and Notice URLs | <img alt="check" src="/assets/images/icons/icon-check-green.png" width="30"> | |
+| Supports Third Party Rendering libraries | | <img alt="check" src="/assets/images/icons/icon-check-green.png" width="30"> |
+
+**Notes:**
+
+- On one hand, using Prebid Cache reduces the amount of data that must be sent to the mobile device -- the body of the creative does not need to be transmitted for bids. On the other hand, though, when a bid wins in the ad server, the body of the creative must be retrieved from the cache.
+- It is possible to mix-and-match implementations within an app. e.g. you could implement instream video with the Bidding-Only approach and other adunits with Prebid-Rendered.
+
+### Prerequisites
+
+The GAM Prebid-Rendered Integration method assumes that you have the following components:
+
+- **Google Ad Manager Account** - A GAM account allows you to manage and serve ads within your mobile app. Within this account you'll need to configure your inventory and create orders for serving ads within your app. This involves defining ad units (spaces within your app where ads will be displayed) and setting up orders and line items to deliver ads to those units. See [Prebid's AdOps Guide](/adops/before-you-start.html) for more information.
+- **Google Mobile Ads (GMA) SDK** - This refers to the software development kit provided by Google. You need to ensure that you have the latest version of the GMA SDK supported by Prebid SDK. This SDK integration is necessary to communicate with the ad server and display ads in your app.
```

Comment:
The current integration is pretty complex and differs from the bidding-only integration. In the Bidding-only - the GMA SDK is a primary integration SDK. However, in the current approach, the Prebid SDK is a primary SDK from the integration perspective. 

The prebid event handler wraps the GMA SDK and Prebid SDK and orchestrates the displaying of the ads frmm the respective source. 

Technically  - publishers don't need to integrate the GMA SDK because it is a dependency of Prebid's event handlers. 

If publisher already has GMA SDK in the app and wants to integrate the GAM Prebid Rendered approach - he or she should conduct a lot of refactoring, substituting the GMA SDK with Prebid SDK in the app layout. 

I think it also should be communicated in the docs. However, it can be done later as a separate scope. You can create a task for me and I'll try to explain in details with some graphical schemas.

Reply:
I'm resolving this comment since there is a section **Migrating banners from a Bidding-Only integration** that describes how to switch to the current integration scenario.